crates/bevy_pbr/src/render/mesh.wgsl

```diff
@@ -8,9 +8,12 @@ struct Vertex {
 #ifdef VERTEX_TANGENTS
     [[location(3)]] tangent: vec4<f32>;
 #endif
+#ifdef VERTEX_COLORS
+    [[location(4)]] color: vec4<f32>;
+#endif
 #ifdef SKINNED
-    [[location(4)]] joint_indices: vec4<u32>;
-    [[location(5)]] joint_weights: vec4<f32>;
+    [[location(5)]] joint_indices: vec4<u32>;
+    [[location(6)]] joint_weights: vec4<f32>;
```

Comment:
I shifted the locations to match the `id`s given in the `MeshVertexAttribute`s in `bevy_render/src/mesh/mesh/mod.rs`.